grails-app/services/org/transmartproject/db/querytool/QueriesResourceService.groovy

```diff
@@ -169,12 +171,12 @@ class QueriesResourceService implements QueriesResource {
     }
 
     @Override
-    List<QueryResult> getQueryResultsByUsername(String username) {
+    List<QueryResultSummary> getQueryResultsSummaryByUsername(String username) {
         def query = QtQueryResultInstance.where {
             queryInstance.userId == username
         }
-        query.findAll()
-    }
+        query.collect { new QueryResultSummaryImplementation(it)}
+}
```

Comment:
btw in intellijJ when you do alt+cmd + L (on linux I am not sure, I think ctrl + alt?) then it will reformat the code nicely for you.